x/participation/client/cli/query_used_allocations_test.go

```diff
@@ -0,0 +1,159 @@
+package cli_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	"github.com/stretchr/testify/require"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/tendermint/spn/testutil/network"
+	"github.com/tendermint/spn/testutil/nullify"
+	"github.com/tendermint/spn/x/participation/client/cli"
+	"github.com/tendermint/spn/x/participation/types"
+)
+
+// Prevent strconv unused error
+var _ = strconv.IntSize
```

Comment:
We have this in the Starport template so that we can import `strconv ` independently if it is used or not in the code scaffolded to avoid import error, so the scaffolding logic is simplified.
Once scaffolded it can always be removed, eventually along with import